planet/order_request.py

```diff
@@ -429,10 +429,12 @@ def toar_tool(scale_factor: Optional[int] = None, ) -> dict:
     return _tool('toar', parameters)
 
 
-def harmonize_tool() -> dict:
+def harmonize_tool(target_sensor: str) -> dict:
```

Comment:
This solution works as-is, so this comment isn't included to reopen the PR as much as a thought exercise:

Is there a way to validate the target_sensor input? AKA check if the value is found within a global set (something like VALID_TARGET_SENSORS) that enumerates the option?

Reply:
Certainly a good point. I'd be happy to add something like `VALID_TARGET_SENSORS` in `planet.specs`.

In `planet.specs` around line 35:
```
VALID_TARGET_SENSORS = ["PS2", "Sentinel-2"]
```

In the test:
```
from planet.specs import VALID_TARGET_SENSORS
...
@pytest.mark.parametrize("target_sensor", VALID_TARGET_SENSORS)
<inset test here>
```

Thoughts on re-opening? Or adding this to another ticket?


_An aside here_: I want to point out that I'm pretty sure case-sensitivity doesn't matter as the target sensors get validated through `planet.specs.get_match()`, which is case-insensitive.

Reply:
yeah that looks great! and no need to reopen. Since you already worked out the solution, just throw it into another ticket that's marked enhancement.

Reply:
Done! #793